assignment/api/src/java/org/sakaiproject/assignment/api/model/Assignment.java

```diff
@@ -251,6 +251,12 @@
     @Column(name = "CONTENT_REVIEW")
     private Boolean contentReview = Boolean.FALSE;
 
+    @Column(name = "OB_ESTIMATE", length = 1)
+    private Boolean obEstimate;
+
+    @Column(name = " ESTIMATE", length = 255)
```

Comment:
ACTUAL_ESTIMATE

Reply:
Updating column OB_ESTIMATE to IS_REQUIRED_ESTIMATE makes clearer what IS_REQUIRED_ESTIMATE and ESTIMATE columns are storing so there's no more need to modify ESTIMATE column.  It also prevents that the variables that store this value through the whole branch might be updated to match this column's name change.